### ISSUE TITLE
[a11y][Mobile] The Product navigation menu cannot be expanded and sometimes the Bento menu is expanded

### ISSUE THREAD

**Issue by abodea** [mozilla/blurts-server#1240]
#### Preconditions
Activate the accessibility on your device.
### Steps to reproduce
1. Install and Launch the app.
2. Access `https://monitor-v2.herokuapp.com/`
3. Try to open the Product Navigation menu(tap to expand).

### Expected behavior
The Product navigation should be expandable while accessibility is activated.
### Actual behavior
The Product navigation menu cannot be expanded and sometimes the Bento menu is expanded
* Android device: `Samsung Galaxy S10+(Android 9), Samsung Galaxy Note 9(Android 8), Samsung Galaxy Tab A(Android 9)`
* Fenix version: `Latest Firefox and Chrome` versions.
### Notes:
For more details please see the attachment.
[log1.txt](https://github.com/mozilla/blurts-server/files/3676836/log1.txt)

**Comment by abodea** [comment]
```Kotlin Device | 0x7903844ea0 | 0000 | RGBA_8888 |   0.0    0.0 1440.0  168.0 |    0 2872 1440 3040 | NavigationBar[6085]#0
2019-10-01 17:19:05.750 5714-5906/? D/WifiPermissionsUtil: canAccessScanResults: pkgName = android, uid = 1000
2019-10-01 17:19:05.878 5714-5714/? D/CustomFrequencyManagerService: releaseDVFSLockLocked : Getting Lock type frm List : DVFS_MIN_LIMIT  frequency : 175500  uid : 1000  pid : 5714  tag : GESTURE_DETECTED@CPU_MIN@49
2019-10-01 17:19:05.886 5714-10529/? I/AfterimageCompensationService: BRIGHTNESS = 172
2019-10-01 17:19:05.988 5714-5880/? D/InputReader: Input event(10): value=1 when=116848.307652
2019-10-01 17:19:05.989 5714-5880/? D/InputReader: Input event(10): value=1 when=116848.307652
2019-10-01 17:19:05.989 5714-5880/? I/InputReader: Touch event's action is 0x0 (deviceType=0) [pCnt=1, s=0.4412 ] when=116848.307652
2019-10-01 17:19:05.991 5714-5748/? D/Sprites: image state surfaceDrawn w:116 h:116
2019-10-01 17:19:06.009 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848307
2019-10-01 17:19:06.015 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848323
2019-10-01 17:19:06.018 24886-24886/? E/TalkBackService: onAccessibilityEvent()mSelectionModeManager=false, mCursorController=false
2019-10-01 17:19:06.020 5238-5238/? I/SurfaceFlinger: Display 0 HWC layers:
        type    |    handle    | flag |  format   |   source crop (l,t,r,b)    |         frame       | name 
    ------------+--------------+------+-----------+----------------------------+---------------------+------
         Device | 0x78e3a87420 | 0002 | RGBx_8888 |   0.0    0.0 1440.0 2726.0 |    0  146 1440 2872 | SurfaceView - org.mozilla.firefox/or[...].gecko.BrowserApp@6dc3805@1[27114]#0
         Device | 0x78e3a86620 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 3040.0 |    0    0 1440 3040 | org.mozilla.firefox/org.mozilla.gecko.BrowserApp[27114]#0
         Device | 0x78e3a87f80 | 0000 | RGBA_8888 |   0.0    0.0    1.0    1.0 |  719 1508  720 1509 | [25701]#0
         Device | 0x78e3a86540 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 2872.0 |    0    0 1440 2872 | [25701]#1
         Device | 0x7903843fc0 | 0000 | RGBA_8888 |   0.0    0.0  157.0  157.0 |    0 1430  157 1587 | [25701]#2
         Device | 0x7903844500 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 3040.0 |    0    0 1440 3040 | [25701]#3
         Device | 0x79038
2019-10-01 17:19:06.020 5238-5238/? I/SurfaceFlinger: 43540 | 0000 | RGBA_8888 |   0.0    0.0 1440.0  142.0 |    0    0 1440  142 | StatusBar[6085]#0
         Device | 0x7903844ea0 | 0000 | RGBA_8888 |   0.0    0.0 1440.0  168.0 |    0 2872 1440 3040 | NavigationBar[6085]#0
         Device | 0x7903e47ba0 | 0000 | RGBA_8888 |   0.0    0.0  116.0  116.0 | 1186  612 1302  728 | Sprite#3
2019-10-01 17:19:06.026 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848340
2019-10-01 17:19:06.043 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848356
2019-10-01 17:19:06.046 5714-5880/? D/InputReader: Input event(10): value=0 when=116848.365798
2019-10-01 17:19:06.046 5714-5880/? D/InputReader: Input event(10): value=0 when=116848.365798
2019-10-01 17:19:06.046 5714-5880/? I/InputReader: Touch event's action is 0x1 (deviceType=0) [pCnt=1, s=] when=116848.365798
2019-10-01 17:19:06.059 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848365
2019-10-01 17:19:06.154 5714-5880/? D/InputReader: Input event(10): value=1 when=116848.474154
2019-10-01 17:19:06.154 5714-5880/? D/InputReader: Input event(10): value=1 when=116848.474154
2019-10-01 17:19:06.154 5714-5880/? I/InputReader: Touch event's action is 0x0 (deviceType=0) [pCnt=1, s=0.4413 ] when=116848.474154
2019-10-01 17:19:06.155 5714-5748/? D/Sprites: image state surfaceDrawn w:116 h:116
2019-10-01 17:19:06.161 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848474
2019-10-01 17:19:06.165 5714-5714/? D/GameManagerService: identifyGamePackage. org.mozilla.firefox
2019-10-01 17:19:06.165 5714-5714/? D/GamePkgDataHelper: getGamePkgData(). org.mozilla.firefox
2019-10-01 17:19:06.165 5714-5714/? D/CustomFrequencyManagerService: acquireDVFSLockLocked : type : DVFS_MIN_LIMIT  frequency : 175500  uid : 1000  pid : 5714  pkgName : GESTURE_DETECTED@CPU_MIN@49
2019-10-01 17:19:06.167 24886-24886/? E/TalkBackService: onAccessibilityEvent()mSelectionModeManager=false, mCursorController=false
2019-10-01 17:19:06.167 24886-24886/? E/TalkBackService: onAccessibilityEvent()mSelectionModeManager=false, mCursorController=false
2019-10-01 17:19:06.168 5238-5238/? I/SurfaceFlinger: Display 0 HWC layers:
        type    |    handle    | flag |  format   |   source crop (l,t,r,b)    |         frame       | name 
    ------------+--------------+------+-----------+----------------------------+---------------------+------
         Device | 0x78e3a87420 | 0002 | RGBx_8888 |   0.0    0.0 1440.0 2726.0 |    0  146 1440 2872 | SurfaceView - org.mozilla.firefox/or[...].gecko.BrowserApp@6dc3805@1[27114]#0
         Device | 0x78e3a86620 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 3040.0 |    0    0 1440 3040 | org.mozilla.firefox/org.mozilla.gecko.BrowserApp[27114]#0
         Device | 0x78e3a87f80 | 0000 | RGBA_8888 |   0.0    0.0    1.0    1.0 |  719 1508  720 1509 | [25701]#0
         Device | 0x78e3a86540 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 2872.0 |    0    0 1440 2872 | [25701]#1
         Device | 0x7903843fc0 | 0000 | RGBA_8888 |   0.0    0.0  157.0  157.0 |    0 1430  157 1587 | [25701]#2
         Device | 0x7903844500 | 0000 | RGBA_8888 |   0.0    0.0 1440.0 3040.0 |    0    0 1440 3040 | [25701]#3
         Device | 0x79038
2019-10-01 17:19:06.168 5238-5238/? I/SurfaceFlinger: 43540 | 0000 | RGBA_8888 |   0.0    0.0 1440.0  142.0 |    0    0 1440  142 | StatusBar[6085]#0
         Device | 0x7903844ea0 | 0000 | RGBA_8888 |   0.0    0.0 1440.0  168.0 |    0 2872 1440 3040 | NavigationBar[6085]#0
         Device | 0x7903e47ba0 | 0000 | RGBA_8888 |   0.0    0.0  116.0  116.0 | 1192  611 1308  727 | Sprite#3
         Device | 0x7903e47200 | 0000 | RGBA_8888 |   0.0    0.0  116.0  116.0 | 1170  602 1286  718 | Sprite#4
2019-10-01 17:19:06.177 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848490
2019-10-01 17:19:06.194 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848507
2019-10-01 17:19:06.210 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848524
2019-10-01 17:19:06.221 5714-5880/? D/InputReader: Input event(10): value=0 when=116848.541051
2019-10-01 17:19:06.221 5714-5880/? D/InputReader: Input event(10): value=0 when=116848.541051
2019-10-01 17:19:06.221 5714-5880/? I/InputReader: Touch event's action is 0x1 (deviceType=0) [pCnt=1, s=] when=116848.541051
2019-10-01 17:19:06.227 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848532
2019-10-01 17:19:06.229 5714-5714/? D/PowerManagerService: [api] userActivity : event: 0 flags: 0 (uid: 1000 pid: 5714) eventTime = 116848541
2019-10-01 17:19:06.233 5714-5714/? E/SamsungTouchExplorer: ACTION_CLICK failed. Dispatching motion events to simulate click.
2019-10-01 17:19:06.235 5714-5714/? D/InputDispatcher: Inject motion (0): action=0x0, eventTime=116848541000000, displayId=0
2019-10-01 17:19:06.235 5714-5714/? D/InputDispatcher: Inject motion (0): action=0x1, eventTime=116848541000000, displayId=0
2019-10-01 17:19:06.235 5714-5879/? I/InputDispatcher: Delivering touch to (6085): action: 0x4, displayId: 0, '456b06d', toolType: 1
2019-10-01 17:19:06.235 5714-5879/? I/InputDispatcher: Delivering touch to (6085): action: 0x4, displayId: 0, 'f334e50', toolType: 1
2019-10-01 17:19:06.235 5714-5879/? I/clp-JNI: Add pinfo.  (channel) 5928798 org.mozilla.firefox/org.mozilla.gecko.BrowserApp (server) (pkg) AppWindowToken{282ba69 token=Token{f0a94f0 ActivityRecord{3ccb933 u0 org.mozilla.firefox/org.mozilla.gecko.BrowserApp t108}}} 27114 10310 (action) 0
2019-10-01 17:19:06.235 5714-5879/? I/clp-JNI: Add pinfo end.
2019-10-01 17:19:06.235 5714-5879/? I/InputDispatcher: Delivering touch to (27114): action: 0x0, displayId: 0, '5928798', toolType: 1
2019-10-01 17:19:06.235 27114-27114/? D/ViewRootImpl@4a1ed4e[BrowserApp]: ViewPostIme pointer 0
2019-10-01 17:19:06.237 5714-5879/? I/clp-JNI: Add pinfo.  (channel) 5928798 org.mozilla.firefox/org.mozilla.gecko.BrowserApp (server) (pkg) AppWindowToken{282ba69 token=Token{f0a94f0 ActivityRecord{3ccb933 u0 org.mozilla.firefox/org.mozilla.gecko.BrowserApp t108}}} 27114 10310 (action) 1
2019-10-01 17:19:06.237 5714-5879/? I/clp-JNI: Add pinfo end.
2019-10-01 17:19:06.237 5714-5879/? I/InputDispatcher: Delivering touch to (27114): action: 0x1, displayId: 0, '5928798', toolType: 1
2019-10-01 17:19:06.237 27114-27114/? D/ViewRootImpl@4a1ed4e[BrowserApp]: ViewPostIme pointer 1
2019-10-01 17:19:06.265 24886-24886/? E/TalkBackService: onAccessibilityEvent()mSelectionModeManager=false, mCursorController=false
2019-10-01 17:19:06.284 5238-5238/? I/SurfaceFlinger: Display 0 HWC layers:

**Comment by lesleyjanenorton** [comment]
Fixed in #1282
